packages/logs/src/logs.ts

```diff
@@ -0,0 +1,120 @@
+import {
+  areCookiesAuthorized,
+  Batch,
+  combine,
+  commonInit,
+  Configuration,
+  Context,
+  ErrorMessage,
+  ErrorObservable,
+  getTimestamp,
+  HttpRequest,
+  InternalMonitoring,
+  mustUseSecureCookie,
+} from '@datadog/browser-core'
+import { buildEnv } from './buildEnv'
+import { Logger, LogsMessage } from './logger'
+import { LoggerSession, startLoggerSession } from './loggerSession'
+import { LogsUserConfiguration } from './logs.entry'
+
+export const startLogs = makeStartLogs((userConfiguration) => {
+  const isCollectingError = userConfiguration.forwardErrorsToLogs !== false
+  const { configuration, internalMonitoring, errorObservable } = commonInit(
+    userConfiguration,
+    buildEnv,
+    isCollectingError
+  )
```

Comment:
IMO we could have an approach
```ts
export function startLogs(...) {
  // base init
  return doStartLogs(...)
}

export function doStartLogs(...) {
  ...
}
```
and unit test `doStartLogs`.
it would remove some complexity introduce by the factory pattern and make more clear what are the input at the different levels.